TROUBLESHOOTING.md

```diff
@@ -0,0 +1,42 @@
+# Troubleshooting
+
+You may find that the Terraform Provider doesn't always provide the most useful error messages.
+Here are some common errors and their workarounds.
+
+## unexpected state 'CART'
+
+```text
+$ terraform destroy
+... snip ...
+cloudbolt_bp_instance.mycbresource: Destroying... [id=/api/v2/resources/service/35/]
+cloudbolt_bp_instance.mycbresource: Still destroying... [id=/api/v2/resources/service/35/, 10s elapsed]
+
+Error: Error waiting for Job (/api/v2/orders/81/) to complete: unexpected state 'CART', wanted target 'SUCCESS'. last error: %!s(<nil>)
+```
+
+This means that the "Delete" resource action requires approval in CloudBolt.
```

Comment:
Should "resource action" be title case?

Reply:
Some of these are questions for @jevanscb

Reply:
```suggestion
This means that the "Delete" resource action Requires Approval in CloudBolt.
```

Reply:
I think we should. Makes it more obvious that it's a CloudBolt property and not some other thing.